### PR TITLE
check before wallet recovery

### PR DESCRIPTION
**Pull request by chauchausoup**
Card: https://github.com/0chain/block-explorer/projects/1#card-73540001

### CODE REVIEW

**Comment by shaktals** [comment]
It looks good!
One thing I was talking with Saswata the other day is about `mainnet`, how we'll probably have dozens and dozens of miners. So making requests to all of them would be carzy (like we are seeing on `testnet0`. What do you think about querying 1 or 2 miners, and if the response is successful just go with it?

As this is not a major operation, just a confirmation, not so important at that.

If the first one or two fails, we could then iterate some 4 or 5 more, check it the response is different?
Not sure how we'd reason about reaching a consensus about it...

**Comment by shaktals** [comment]
One example on how to do it is shown on the [getInformationFromRandomSharder](https://github.com/0chain/js-client-sdk/blob/master/index.js#L1149) code.

**Comment by chauchausoup** [comment]
> One example on how to do it is shown on the [getInformationFromRandomSharder](https://github.com/0chain/js-client-sdk/blob/master/index.js#L1149) code.

File: utils.js [hunk 1]
```diff
@@ -335,6 +344,7 @@ module.exports = {
             const promises = urls.map(url => self.getReq(url, params));
             let percentage = Math.ceil((promises.length * consensusPercentage) / 100);
 
+            console.log(promises,"prom")
```
Comment (shaktals) [inline utils.js:347]:
```suggestion
```

**Review by shaktals — APPROVED** [review]
@esliagolli can you also test this out before we merge it to master? One easy way I remember lately is to use the [npm link](https://docs.npmjs.com/cli/v7/commands/npm-link) feature. You can just check out the branch locally, and local explorer will use that version of the lib.

@chauchausoup we need to upgrade the package version too.

**Review by shaktals — APPROVED** [review]

**Pull request merged**